cfy_cluster_setup/logger.py

```diff
@@ -5,15 +5,16 @@
 
 FORMAT_MSG = '%(name)s - %(levelname)s - %(message)s'
 
+
 def setup_logger(verbose):
     logger = logging.getLogger()
     logger.setLevel(logging.DEBUG)
 
     # paramiko.transport, invoke, and fabric are very verbose
-    if not verbose:
-        logging.getLogger('paramiko.transport').setLevel(logging.WARNING)
-        logging.getLogger('invoke').setLevel(logging.WARNING)
-        logging.getLogger('fabric').setLevel(logging.WARNING)
+    packages_log_level = logging.INFO if verbose else logging.WARNING
```

Comment:
The DEBUG level is very (very) verbose in these packages.

Reply:
Following the comment on Slack, I changed the log level from `logging.DEBUG` to `logging.INFO` when `verbose` because the DEBUG level is too verbose (in my opinion).